tools/generate-imports.mjs

```diff
@@ -62,7 +67,62 @@ function expandPaths(paths) {
     })
     .reduce((x, y) => x.concat(y));
 }
+/**
+ * @param {string} manager
+ * @param {string} filePath
+ * @returns {Promise<string>}
+ */
+async function getFileHash(manager, filePath) {
+  try {
+    const hash = await hasha.fromFile(
+      `lib/modules/manager/${manager}/${filePath}`,
+      options
+    );
+    return hash;
+  } catch (err) {
+    throw new Error(
+      `ERROR: Unable to generate hash for manager/${manager}/${filePath}`
+    );
+  }
+}
+/**
+ *
+ * @param {string} managerName
+ * @returns {Promise<string>}
+ */
+export async function getManagerHash(managerName) {
+  try {
+    let hashes = [];
+    let files = await fs.readdir(`lib/modules/manager/${managerName}`);
```

Comment:
Can we instead get a recursive list of files and filter for *.snap and *.spec.ts ?

Reply:
i think we can use `glob` for this.